src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineStandardStaging.java

```diff
@@ -80,6 +83,13 @@ public void stageStandard(StageStandardConfiguration config) throws AppEngineExc
 
       cloudSdk.runAppCfgCommand(arguments);
 
+      //TODO : Move this fix up the chain (appcfg)
+      if (config.getRuntime() != null && config.getRuntime().equals("java")) {
```

Comment:
Good question, but I don't know. In any case, this is more of an implementation detail of staging and deployment in AppCfg and gcloud.